tests/test_handler_mlflow.py

```diff
@@ -138,10 +143,12 @@ def _update_metric(engine):
                 state_attributes=["test"],
                 experiment_param=experiment_param,
                 artifacts=[artifact_path],
-                close_on_complete=True,
+                close_on_complete=False,
```

Comment:
Not sure if this will cause other mlflow cases error. Let's run the test and see the result.

Reply:
All test passed.